docs/admin-manual/configuration-macros.rst

```diff
@@ -6534,6 +6534,17 @@ These settings affect the *condor_starter*.
     to the Singularity command line. This can be an expression evaluted in the context of the
     slot ad and the job ad, where the slot ad is the "MY.", and the job ad is the "TARGET.".
 
+:macro-def:`USE_DEFAULT_CONTAINER[STARTER]`
+    A boolean value or classad expression evaluting to boolean in the context of the Slot
+    ad (the MY.) and the job ad (the TARGET.).  When try, a vanilla universe job that
+    does not request a container will be put into the container image specified By
+    the parameter :macro:`DEFAULT_CONTAINER_IMAGE`
+
+:macro-def:`USE_DEFAULT_CONTAINER[STARTER]`
```

Comment:
That should be `DEFAULT_CONTAINER_IMAGE` ;-).

Reply:
Ooops -- thanks for the heads-up.  Will get it fixed right away.